pyms/flask/services/requests.py

```diff
@@ -7,17 +7,32 @@
 from flask import request
 from requests.adapters import HTTPAdapter, Response
 from urllib3.util.retry import Retry
+try:
+    from prometheus_client import Counter, Histogram
+except ModuleNotFoundError:  # pragma: no cover
+    Counter = None
+    Counter = Histogram
 
 from pyms.constants import LOGGER_NAME
-from pyms.flask.services.driver import DriverService
+from pyms.flask.services.driver import DriverService, get_service_name
 from pyms.flask.services.tracer import inject_span_in_headers
+from pyms.config.conf import get_conf
 
 logger = logging.getLogger(LOGGER_NAME)
 
 DEFAULT_RETRIES = 3
 
 DEFAULT_STATUS_RETRIES = (500, 502, 504)
 
+METRICS_CONFIG = get_conf(service=get_service_name(
+    service="metrics"), empty_init=True)
+
+REQUESTS_COUNT = Counter("http_server_responses_count", "Python requests count", [
```

Comment:
If Counter = None, you'll get "None type is not callabe"

Reply:
You're right, I thought I've tested it enabling and disabling the `metrics` section on the config, but I still had prometheus_client installed.

Reply:
Changed.

Reply:
Uh, when executing the tests, I've seen that doing `import pdb;pdb.set_trace()` on line 29 and executing in there:
```text
get_conf(service=get_service_name(service="metrics"), empty_init=True)
get_conf(service=get_service_name(service="requests"), empty_init=True)
```
Or any service, it always returns a empty value. So `REQUESTS_COUND` and `REQUESTS_LATENCY` are never initialized and fail later, on `observe_requests`. I would say that the configuration is incorrect, but I can't see anything. Any idea?

Reply:
I've changed it already.